frontend/src/tests/Dashboard.test.js

```diff
@@ -4,9 +4,19 @@ import Dashboard from "../Dashboard";
 import LOCALIZE from "../text_resources";
 
 describe("renders title and description", () => {
+  localStorage.setItem("first_name", "Hello");
```

Comment:
Is there a way to do this test without localstorage?

Reply:
I don't need that anymore, since I'm not using the local storage now.
See commit #https://github.com/code-for-canada/project-thundercat/pull/360/commits/25d56203b5b008cfb631d43afb1d801382761c43 for updated tests.